install.sh

```diff
@@ -10,6 +10,29 @@ case $OSTYPE in
     ;;
 esac
 
+BACKUP_FILE=$CONFIG_FILE.bak
+
+if [ -e $HOME/$BACKUP_FILE ]; then
+  echo "Backup file already exist. Make sure to backup your .bashrc before running this installation." >&2
+     while true
+    do
+      read -p "Would you like to overwrite the existing backup? this will delete your existing backup file [Y/N] " RESP
```

Comment:
Secondly, it would be great if you could use the same `read` semantics as in the other use cases in this file (take a look at https://github.com/Bash-it/bash-it/blob/master/install.sh#L43 for an example):

* Use the same parameters for read to expect one character.
* Indicate a default value by making one of the options (Y/N) uppercase and the other lowercase, e.g. _y/N_